launch/bayer_rggb_image_proc.launch

```diff
@@ -1,16 +1,17 @@
+<?xml version="1.0"?>
 <launch>
   <arg name="nodelet_manager_name" value="nodelet_manager" />
-  <arg name="camera_name" value="camera" />
+  <arg name="camera_name" default="camera" />
 
   <node pkg="nodelet" type="nodelet" name="$(arg nodelet_manager_name)" args="manager" output="screen" />
-  
+
   <node pkg="nodelet" type="nodelet" name="ueye_cam_nodelet"
         args="load ueye_cam/ueye_cam_nodelet $(arg nodelet_manager_name)">
     <param name="camera_name" type="str" value="$(arg camera_name)" /> <!-- == namespace for topics and services -->
     <param name="camera_topic" type="str" value="image_raw" />
     <param name="camera_id" type="int" value="0" /> <!-- 0 = any camera; 1+: camera ID -->
-    <param name="camera_intrinsics_file" type="string" value="" /> <!-- default: ~/.ros/camera_info/<camera_name>.yaml -->
-    <param name="camera_parameters_file" type="string" value="" /> <!-- default: ~/.ros/camera_conf/<camera_name>.ini -->
+    <param name="camera_intrinsics_file" type="string" value="$(env HOME)/.ros/camera_info/$(arg camera_name).yaml" />
+    <param name="camera_parameters_file" type="string" value="$(env HOME)/.ros/camera_parameters_file/$(arg camera_name).ini" />
```

Comment:
please do not rename camera_conf into camera_parameters_file folder, as this may break backward compatibility for people using the previous default path

Reply:
done